src/VscPreviewExtension/Microsoft.DocAsCode.Dfm.VscPreview/DfmProcess.cs

```diff
@@ -14,21 +14,28 @@ internal class DfmProcess
         static void Main(string[] args)
         {
             DfmJsonTokenTreeServiceProvider dfmJsonTokenTreeServiceProvider = new DfmJsonTokenTreeServiceProvider();
-            IMarkdownService dfmMarkdownService = dfmJsonTokenTreeServiceProvider.CreateMarkdownService(new MarkdownServiceParameters());
+            IMarkdownService dfmMarkdownService =
+                dfmJsonTokenTreeServiceProvider.CreateMarkdownService(new MarkdownServiceParameters());
             while (true)
             {
                 try
                 {
                     string command = Console.ReadLine();
+                    string result;
                     switch (command.ToLower().Trim())
                     {
                         case "exit":
                             return;
-                        case "dfmmarkup":
-                            DfmMarkupReceiveContent();
+                        case "docfxpreview":
+                            string baseDir = Console.ReadLine();
+                            string relativePath = Console.ReadLine();
+                            result = DocfxProcessor.DocfxProcess(baseDir, relativePath, GetMarkdownContent());
+                            SendWithEndCode(result);
                             break;
-                        case "jsonmarkup":
-                            JsonMarkupReceiveContent(dfmMarkdownService);
+                        case "tokentreepreview":
+                            result = TokenTreeProcessor.TokenTreePreview(dfmMarkdownService, GetMarkdownContent());
```

Comment:
they not the same Service, but to be consistent, I should't pass dfmMarkdownService to tokenTreeProcessor.

---
In reply to: [102653359](https://github.com/dotnet/docfx/pull/1313#discussion_r102653359) [](ancestors = 102653359)